gnucash/report/reports/standard/income-gst-statement.scm

```diff
@@ -159,20 +159,17 @@ for taxes paid on expenses, and type LIABILITY for taxes collected on sales.")
    options
    (gnc:make-multichoice-callback-option
     pagename-format (N_ "Report format")
-    "a" (G_ "Report Format") 'default
-    (list (vector 'default
-                  (G_ "Default Format")
-                  (G_ "Default Format"))
-          (vector 'au-bas
-                  (G_ "Australia BAS")
-                  (G_ "Australia Business Activity Statement. Specify sales, \
-purchase and tax accounts."))
-          (vector 'uk-vat
-                  (G_ "UK VAT Return")
-                  (G_ "UK VAT Return. Specify sales, purchase and tax \
+    "a" 
+    (G_ "For Australia Business Activity Statement. Specify sales, \
+purchase and tax accounts\nFor UK VAT Return. Specify sales, purchase and tax \
 accounts. EU rules may be used. Denote EU VAT accounts *EUVAT* in \
 account description, and denote EU goods sales and purchases accounts \
-with *EUGOODS* in the account description."))) #f
+with *EUGOODS* in the account description.")
+    'default
+    (list (vector 'default (G_ "Default Format"))
+          (vector 'au-bas (G_ "Australia BAS"))
+          (vector 'uk-vat (G_ "UK VAT Return")))
+     #f
```

Comment:
As mentioned in an earlier comment, perhaps add some formatting to the tool tip. Without list indicators it's not so easy to read.

Also  out of curiosity, I see you are escaping newlines and adding "\n" in several places. Can this be simplified to not escape newlines and just put them where you consider them appropriate ?

For example I **think** these are equivalent:
```
"One\n\
two\n\
three"

"One
two
three"

```

The second one is definitely more readable.